README.md

```diff
@@ -27,9 +27,9 @@ EQL provides a data format for transmitting and storing encrypted data & indexes
 
 ## Installation
 
-The simplest and fastest way to get up and running with EQL from scratch is to execute the install SQL file directly in your database.
+The simplest and fastest way to get up and running with EQL is to execute the install SQL file directly in your database.
 
-1. Download the [cipherstash-encrypt-dsl.sql](./release/cipherstash-encrypt-dsl.sql) file
+1. Download [cipherstash-encrypt-dsl.sql](./release/cipherstash-encrypt-dsl.sql).
```

Comment:
Update to use the release artifact:

```suggestion
1. Get the latest EQL install script:
   ```bash
    curl -sLo cipherstash-eql.sql https://github.com/cipherstash/encrypt-query-language/releases/latest/download/cipherstash-eql.sql
   ```
```